interface/patient_file/report/custom_report.php

```diff
@@ -188,27 +188,6 @@ function zip_content($source, $destination, $content = '', $create = true)
       // by div of navigateLink class. Specifically used for CAMOS, but
```

Comment:
The table isn't correctly distributing its cells.
![image](https://user-images.githubusercontent.com/46497068/91070967-8f340f00-e605-11ea-9702-46b5692f8ddd.png)

Reply:
The patient data table was generated by `openemr/library/options.inc.php`, this file has a `display_layout_rows ` function to build the `tr` and `td`.

I worry about if I modify the `display_layout_rows function`, will cause another issue.

Reply:
@bradymiller